src/averageAges.js

```diff
@@ -15,11 +15,20 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(el => el.sex === 'm');
+  const yearsDifference = el => el.died - el.born;
+  const totalYears = (acc, val) => acc + val;
```

Comment:
you still creating this in every function again and again, move it into separated function and reuse everywhere. If you don't understand smth feel free to ask in the chat